test/test_seawater_density.jl

```diff
@@ -31,7 +31,9 @@ function error_non_Boussinesq(arch, FT)
 
     grid = RectilinearGrid(arch, FT, size=(3, 3, 3), extent=(1, 1, 1))
     buoyancy = SeawaterBuoyancy(FT)
-    model = NonhydrostaticModel(grid; buoyancy, tracers)
+    model = NonhydrostaticModel(grid;
+                                 buoyancy,
+                                 tracers)
```

Comment:
```suggestion
    model = NonhydrostaticModel(grid; buoyancy, tracers)
```

i think the wrong prompt was given, it is trying to put everything on multiple lines.